src/Pages/SubsitesVirtualPage.php

```diff
@@ -21,139 +19,68 @@
-                $linkToContentLabelField = LiteralField::create('VirtualPageContentLinkLabel', $linkToContent),
-                'Title'
+                DropdownField::create(
+                    'CopyContentFromID_SubsiteID',
+                    _t(__CLASS__ . '.SubsiteField', 'Subsite'),
+                    $subsites->map('ID', 'Title')
+                )->addExtraClass('subsitestreedropdownfield-chooser no-change-track'),
+                'CopyContentFromID'
             );
-        }
 
-
-        $fields->addFieldToTab(
-            'Root.Main',
-            TextField::create(
-                'CustomMetaTitle',
-                $this->fieldLabel('CustomMetaTitle')
-            )->setDescription(_t(__CLASS__ . '.OverrideNote', 'Overrides inherited value from the source')),
-            'MetaTitle'
-        );
-        $fields->addFieldToTab(
-            'Root.Main',
-            TextareaField::create(
-                'CustomMetaKeywords',
-                $this->fieldLabel('CustomMetaKeywords')
-            )->setDescription(_t(__CLASS__ . '.OverrideNote', 'Overrides inherited value from the source')),
-            'MetaKeywords'
-        );
-        $fields->addFieldToTab(
-            'Root.Main',
-            TextareaField::create(
-                'CustomMetaDescription',
-                $this->fieldLabel('CustomMetaDescription')
-            )->setDescription(_t(__CLASS__ . '.OverrideNote', 'Overrides inherited value from the source')),
-            'MetaDescription'
-        );
-        $fields->addFieldToTab(
-            'Root.Main',
-            TextField::create(
-                'CustomExtraMeta',
-                $this->fieldLabel('CustomExtraMeta')
-            )->setDescription(_t(__CLASS__ . '.OverrideNote', 'Overrides inherited value from the source')),
-            'ExtraMeta'
-        );
-
-        return $fields;
-    }
-
-    public function fieldLabels($includerelations = true)
-    {
-        $labels = parent::fieldLabels($includerelations);
-        $labels['CustomMetaTitle'] = _t('SilverStripe\\Subsites\\Model\\Subsite.CustomMetaTitle', 'Title');
-        $labels['CustomMetaKeywords'] = _t(
-            'SilverStripe\\Subsites\\Model\\Subsite.CustomMetaKeywords',
-            'Keywords'
-        );
-        $labels['CustomMetaDescription'] = _t(
-            'SilverStripe\\Subsites\\Model\\Subsite.CustomMetaDescription',
-            'Description'
-        );
-        $labels['CustomExtraMeta'] = _t(
-            'SilverStripe\\Subsites\\Model\\Subsite.CustomExtraMeta',
-            'Custom Meta Tags'
-        );
-
-        return $labels;
+            // Setup the linking to the original page.
+            $pageSelectionField = SubsitesTreeDropdownField::create(
+                'CopyContentFromID',
+                _t('SilverStripe\\CMS\\Model\\VirtualPage.CHOOSE', 'Linked Page'),
+                SiteTree::class,
+                'ID',
+                'MenuTitle'
+            );
```

Comment:
This field is moved to the top which is right next to the subsite selector field and matches where it sits in the normal virtual page.